app/api/__init__.py

```diff
@@ -216,6 +216,7 @@
 api.route(SponsorRelationship, 'sponsor_event', '/sponsors/<int:id>/relationships/event')
 
 # tracks
+api.route(TrackListPost, 'track_list_post', '/tracks')
 api.route(TrackList, 'track_list', '/tracks', '/events/<int:event_id>/tracks',
```

Comment:
TrackListPost allows GET and POST both on `/tracks`, right and track_organiser access is checked in before_post so we can make GET request here also. Why do we need `/tracks` under TrackList also?